src/utils.rs

```diff
@@ -3,7 +3,7 @@ use crate::IcedContext;
 use bevy_math::Vec2;
 use bevy_window::Window;
 
-pub(crate) fn process_cursor_position(
```

Comment:
Why remove the `(crate)` modifiers?

Reply:
The module is already private, it is a bit misleading having pub(crate) inside private module.